examples/advanced/federated-statistics/hierarchical_stats.ipynb

```diff
@@ -0,0 +1,236 @@
+    "runner.run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "51be8a9a-b1b8-413c-abab-5cbd7e191a0d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "**Run Job using Simulator CLI**\n",
+    "\n",
+    "From a **terminal** one can also the following equivallent CLI\n",
+    "\n",
+    "```\n",
+    "cd NVFlare/examples/advanced/hierarchical_stats\n",
+    "nvflare simulator hierarchical_stats/jobs/hierarchical_stats -w /tmp/nvflare/hierarchical_stats/ -n 7 -t 7 -c university-1,university-2,university-3,university-4,university-5,university-6,university-7\n",
+    "\n",
+    "```\n",
+    "\n",
+    "assuming the nvflare is installed from a **terminal**. doing pip install from the notebook cell directory with bash command (! or %%bash) may or may not work depending on which python runtime kernel selected. Also %pip install or %pip install from notebook cell doesn't register the console_scripts in the PATH.   \n",
+    "\n",
+    "\n",
+    "## Examine the result\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "45bf6e9a-3265-4e45-8b06-c8e543605f21",
+   "metadata": {},
+   "source": [
+    "\n",
+    "The results are stored in workspace \"/tmp/nvflare\"\n",
+    "```\n",
+    "/tmp/nvflare/hierarchical_stats/server/simulate_job/statistics/hierarchical_stats.json\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "112a7dd0-45d9-42ea-98b2-f72a3bbccf48",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "cat /tmp/nvflare/hierarchical_stats/server/simulate_job/statistics/hierarchical_stats.json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3cd042db-6ce0-4e37-bcbe-d96051e4d164",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Visualization\n",
+    "We can visualize the results easly via the visualizaiton notebook. Before we do that, we need to copy the data to the notebook directory \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a3c89693-37b9-450c-85dd-8a2d78fee3fa",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "! cp /tmp/nvflare/hierarchical_stats/server/simulate_job/statistics/hierarchical_stats.json hierarchical_stats/demo/."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5c6f632-3326-4236-902e-8c0965688d85",
+   "metadata": {},
+   "source": [
+    "now we can visualize via the [visualization notebook](hierarchical_stats/demo/visualization.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52d2a52f-8a8d-45ef-8a50-ddbb4ed2f2c6",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "We are not quite done yet. What if you prefer to use python API instead CLI to run jobs. Lets do that in this section"
+   ]
```

Comment:
this line confuses me a bit, what "section" are we referring to here?

Reply:
Actually that line got copied from the other notebook I referred. Removed it now as I don't see it relevant here. Thanks.